python/paddle/fluid/tests/unittests/test_imperative_save_load.py

```diff
@@ -363,8 +363,8 @@ def testLoadAndSetVarBase(self):
             if isinstance(adam._learning_rate, LearningRateDecay):
                 adam._learning_rate.step_num = 0
 
-            load_dict = fluid.load_optimizer("./test_dy")
-            adam.set_dict(load_dict)
+            para_state_dict, opti_state_dict = fluid.load_dygraph("./test_dy")
```

Comment:
how about add an test to test if the program can work properly while loaded optimizer is None